lua/neogit/lib/git/log.lua

```diff
@@ -515,4 +515,8 @@ function M.reflog_message(skip)
     .call_sync({ ignore_error = true }).stdout
 end
 
+function M.abbreviated_size()
+  return string.len(cli.log.format("%h").max_count(1).call_sync().stdout[1])
```

Comment:
```suggestion
  return string.len(cli.log.format("%h").max_count(1).call({ hidden = true }).stdout[1])
```

`hidden = true` will prevent this from showing up in the git command history buffer.